src/main/java/dog/pawbook/ui/EntityCard.java

```diff
@@ -38,13 +45,31 @@
     private Label id;
     @FXML
     private FlowPane tags;
+    @FXML
+    private Label staticId;
+    @FXML
+    private ImageView entityImage;
+
+
 
     /**
      * Creates a {@code EntityCard} with the given {@code Entity} and index to display.
      */
     public EntityCard(Entity entity, int displayedId) {
         super(FXML);
         this.entity = entity;
+
+        this.entityImage.setFitHeight(50);
+        this.entityImage.setFitWidth(50);
```

Comment:
is there a way to specify this in fxml? I think it is generally better if parameters that don't need to be dynamically set be left out of the code

Reply:
As of now, I couldn't find. Will try to implement this in the next PR if possible.